### ISSUE TITLE
P1 a11y bug - episode list - focus styling should be applied the same as hover styles as specified by UX

### ISSUE THREAD

**Issue by jroebu14** [BBC-archive/psammead#4036]
**Describe the bug**
Focus styling should be applied the same as hover styles as specified by UX (with the addition of the browser default outline as is currently applied). Note browser default focus styles do not always meet colour contrast standards. https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5f2035d795f41b8a78b9ea12 

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
CC @greenc05

**Comment by aeroplanejane** [comment]
This is a p1 because no focus styles have been applied to the component. 
Users are only getting the browser default such as a dotted outline in FF. If you use the tab key to move through the links you will see this. The browser default for this that doesn’t always meet contrast and this is why this is a P1.

**Comment by aeroplanejane** [comment]
Reference: https://www.bbc.co.uk/accessibility/forproducts/guides/mobile/visible-focus
“Do not depend on browser default focus states for hover, focus, or touch, as they are inconsistently implemented across browsers, are not always obvious, and can lack sufficient contrast depending on the design.”

**Comment by LilyL0u** [comment]
On hover, the icon now fills with colour and inverts. The title and description are underlined, with the title getting a bold underline. Fixed.

**Comment by greenc05** [comment]
Think you meant on focus @LilyL0u?

**Comment by LilyL0u** [comment]
Yes, I meant focus!
